target/linux/layerscape/armv8_64b/target.mk

```diff
@@ -0,0 +1,15 @@
+#
+# Copyright (C) 2016 Jiang Yutang <jiangyutang1978@gmail.com>
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+ARCH:=aarch64
```

Comment:
Actually, ls1012a/ls1088a/ls1043a is cortex A53 cpu while ls1046a/ls2088a is cortex A57 cpu.